.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 120
+ignore = ['D100','D101','D102','D103','D104','D401','D400','D107','W503','D200']
```

Comment:
Why did you need to ignore all these rules?

Reply:
I did not add that many, if you look in .pre-commit-config.yaml you were already skipping D102,D104,D401,D400,D107,W503, so I just added the following to the flake8 config so that users running in their editor get the same rules applied as pre-commit is using.

D100:Missing docstring in public module
D101:Missing docstring in public class
D103:Missing docstring in public function
D200:One-line docstring should fit on one line with quotes

All of these are doc string related and I do not feel qualifies to add all the missing doc strings.  In addition I already fixed hundreds of errors since the repo is clearly never running pre-commit checks.  I would suggest merging my chnages and making a seperate PR to clear up all skipped errors.